ios/brave-ios/Sources/Brave/WebFilters/DownloadResourceInterface.swift

```diff
@@ -66,11 +57,17 @@ extension DownloadResourceInterface {
   ///
   /// - Note: Returns nil if the cache folder does not exist
   var createdCacheFolderURL: URL? {
-    guard let folderURL = Self.cacheFolderDirectory.url else { return nil }
-    let cacheFolderURL = folderURL.appendingPathComponent(cacheFolderName)
+    guard
+      let folderURL = try? AsyncFileManager.default.url(
```

Comment:
Is this creating the folder? It shouldn't really but if it does, it's not a big deal.

Reply:
It is not